homeassistant/components/media_player/sonos.py

```diff
@@ -973,6 +995,7 @@ def turn_off(self):
     @soco_coordinator
     def media_play(self):
         """Send play command."""
+        self._player.play_mode = self.get_play_mode()
```

Comment:
**Question:**
Can these lines be removed? If we don't set play mode here the device will remember it's current mode, right?

Reply:
You are right. Totally unnecessary